packages/@okta/vuepress-site/docs/guides/oie-embedded-common-download-setup-app/main/index.md

```diff
@@ -22,33 +22,32 @@ Before you download the SDK, Widget, and sample apps, you need the following:
 
 ## Configure the SDK, Sign-In Widget, and sample app
 
-Before you integrate either the SDK or the Widget into your app, you need to understand the configuration settings that you need to add to initialize the SDK and the widget (if you decide to also use the Sign-In Widget).
+Before you integrate either the SDK or the Widget into your app, you need to understand the configuration settings required to initialize the SDK and/or the Sign-In Widget.
 
 ### Configuration settings
 
 #### Issuer
 
-Use the Issuer URI named "default" from your Custom Authorization Server. To find this value, go to **Security** > **API** in the Admin Console. On the **Authorization Servers** tab, the **default** Custom Authorization Server's Issuer URI appears in the list (for example, `https://${yourOktaDomain}/oauth2/default`).
+If you're getting started with your first app or if you're running an Okta sample app, use the Issuer URI of your default Custom Authorization Server. To find this value:
 
-There are two main types of authorization servers: **Org** and **Custom**:
+1. In the Admin Console, go to **Security** > **API**.
+2. On the **Authorization Servers** tab, use the **Issuer URI** value from the **default** Custom Authorization Server row (for example, `https://${yourOktaDomain}/oauth2/default`).
 
-* **Org**: If you want to use the Org Authorization Server, use the following URL: `https://${yourOktaDomain}` (for example, `https://dev-example.okta.com`).
-* **Custom**: You can either use the **default** Custom Authorization Server or create your own.
-  * **default**: The **default** Custom Authorization Server is predefined and automatically created when you created the org.<br>
-  **URI format**: `https://${yourOktaDomain}/oauth2/default`
+There are two main types of authorization servers in Okta: **Org** and **Custom**. See [Authorization Servers](/docs/concepts/auth-servers/#available-authorization-server-types) and [Difference between Okta as an Authorization Server vs Custom Authorization Server](https://support.okta.com/help/s/article/Difference-Between-Okta-as-An-Authorization-Server-vs-Custom-Authorization-Server?language=en_US) for more information on which Authorization Server to use.
 
-      > **Note:** If you are just getting started and want to run the sample app, use the **default** Custom Authorization Server URI.
+If you are using either the Org or the Custom Authorization Servers, your Issuer URI should be in the following format:
 
-  * **Create your own**: You can also create your own Custom Authorization Server.<br>
-  **URI format**: `https://${yourOktaDomain}/oauth2/${authServerId}`
+* **Org**: If you want to use the Org Authorization Server, use the following URI format: `https://${yourOktaDomain}` (for example, `https://dev-example.okta.com`).
+* **Custom**: You can either use the default Custom Authorization Server or create your own.
```

Comment:
```suggestion
* **Custom**: You can use either the default Custom Authorization Server or create your own.
```

Reply:
talked to Christine, we will keep the original